code/game/turfs/space/space.dm

```diff
@@ -21,6 +21,9 @@
 	if(config)
 		update_starlight() //MC will initialize all the space turfs that get created before config
 
+/turf/space/Destroy()
+	return QDEL_HINT_LETMELIVE

```

Comment:
tested this out and it didn't generate anything strange; light was produced, there was a tile over top of space...couldn't notice anything odd from an ICly standpoint--didn't notice any runtimes in the console or the GC profiler having a fit either.